src/Tool-Base/PharoShortcuts.class.st

```diff
@@ -479,7 +479,7 @@ PharoShortcuts >> selectForwardLineShortcut [
 
 { #category : #'keymaps - editor selection' }
 PharoShortcuts >> selectForwardWordShortcut [
-	^ Character arrowRight shift meta
+	^ Character arrowRight shift ctrl
```

Comment:
I'm not sure that this change will work on Mac.

Reply:
Changed + Added comment